/**:
  ros__parameters:
    frame_id: rplidar_frame
    # serial - specify baudrate and serial port
    # tcp - specify TCP address and port
    channel_type: "serial"
    tcp_ip: "192.168.0.7"
    tcp_port: 20108
    serial_baudrate: 256000
    serial_port: "/dev/rplidar"

    raw_enabled: true # Enable/Disable raw scan publish
    legacy_mode: false # legacy mode cuts off rays at the beginning and end of the scan msg if the values are inf

    inverted: false
    angle_compensate: false

    # Sensitivity: optimized for longer ranger, better sensitivity but weak environment elimination - indoor environment
    # Boost: optimized for sample rate
    # Stability: for light elimination performance, but shorter range and lower sample rate - outdoor environment
    scan_mode: "Stability"

    filter:
      enabled: true #Enable/Disable filtering and publisher
      min_range: 0.3
      check_distance: 10.0
      scan_search_area: 15
      minimal_number_of_close_samples: 8
      minimal_distance_for_acceptance_samples: 0.1
